cupy/_core/_scalar.pyx

```diff
@@ -61,8 +61,8 @@ cdef object _numpy_float32 = numpy.float32
 cdef object _numpy_float64 = numpy.float64
 cdef object _numpy_complex64 = numpy.complex64
 cdef object _numpy_complex128 = numpy.complex128
-cdef object _numpy_float_ = numpy.float_
-cdef object _numpy_complex_ = numpy.complex_
+cdef object _numpy_float_ = numpy.float64
+cdef object _numpy_complex_ = numpy.complex128
```

Comment:
Just to check, they were previously aliases for `double` and `cdouble`. I think that doesn't matter here but just in case you were wanting a straight alias swap